cvat-ui/src/actions/requests-async-actions.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { ThunkAction } from 'utils/redux';
+import { CombinedState, RequestsQuery, StorageLocation } from 'reducers';
+import {
+    getCore, RQStatus, Request, Project, Task, Job,
+} from 'cvat-core-wrapper';
+import { exportActions, listenExportBackupAsync, listenExportDatasetAsync } from './export-actions';
+import { RequestInstanceType, requestsActions } from './requests-actions';
+import { importActions, listenImportDatasetAsync } from './import-actions';
+
+const core = getCore();
+
+export interface RequestParams {
+    id: string;
+    type: string;
+    instance?: Project | Task | Job;
+    location?: StorageLocation;
+}
+
+export function getRequestsAsync(query: RequestsQuery): ThunkAction {
+    return async (dispatch, getState): Promise<void> => {
+        dispatch(requestsActions.getRequests(query));
+
+        const state: CombinedState = getState();
+
+        try {
+            const requests = await core.requests.list();
+            dispatch(requestsActions.getRequestsSuccess(requests));
+
+            requests
+                .filter((request: Request) => [RQStatus.STARTED, RQStatus.QUEUED].includes(request.status))
+                .forEach((request: Request): void => {
+                    const {
+                        id: rqID,
+                        operation: {
+                            type, target, format, taskID, projectID, jobID,
+                        },
+                    } = request;
+                    let instance: RequestInstanceType | null = null;
+
+                    const [operationType, operationTarget] = type.split(':');
+                    if (target === 'task') {
+                        instance = { id: taskID as number, type: target };
+                    } else if (target === 'job') {
+                        instance = { id: jobID as number, type: target };
+                    } else if (target === 'project') {
+                        instance = { id: projectID as number, type: target };
+                    }
+
+                    if (operationType === 'export') {
+                        if (operationTarget === 'backup') {
+                            if (!(state.export.tasks.backup.current?.[(instance as RequestInstanceType).id] ||
+                                state.export.projects.backup.current?.[(instance as RequestInstanceType).id])
+                            ) {
+                                dispatch(exportActions.exportBackup(instance as RequestInstanceType));
```

Comment:
We already discussed it, so, what is the reason to call here:

- exportActions.exportDataset
- importActions.importDataset
- exportActions.exportBackup

And generally store import/export as activities. 
Only to lock some controls on UI?